### ISSUE TITLE
http://mirageos.org redirects to mirage.io but https://mirageos.org does not

### ISSUE THREAD

**Issue by kit-ty-kate** [mirage/mirage-www#773]
Not sure if it’s intended.

**Comment by TheLortex** [comment]
Not really, but it's expected. The website doesn't look at which hostname is used. The redirection happens unconditionally when a request is made on port 80. We could add a permanent redirection from mirageos.org to mirage.io by looking at the request's host field.

**Comment by avsm** [comment]
Sounds like we should do the opposite :-) Redirect mirageos.io to mirageos.org, following the discussion in #850; thanks @gm0stache
